show-location-history/README.md

```diff
@@ -0,0 +1,35 @@
+# Show location history
+
+Display your location history on the map.
+
+![Image of show location history](show-location-history.png)
+
+## Use case
+
+You can track device location history and display it as lines and points on the map. The history can be used to visualize how the user moved through the world, to retrace their steps, or to create new feature geometry. An unmapped trail, for example, could be added to the map using this technique.
+
+## How to use the sample
+
+Tap the floating action button to start tracking your location, which will appear as points on the map. A line will connect the points for easier visualization. Tap the button again to stop updating the location history. This sample uses a simulated data source. To track a user's real position, use the `DefaultLocationDataSource` instead.
+
+## How it works
+
+1. Create a graphics overlay to show each point and another graphics overlay to display the route polyline.
+2. Create a `SimulatedLocationDataSource` and initialize it with a polyline. Start the `SimulatedLocationDataSource` to begin receiving location updates.
+3. Use `LocationChanged` on the `simulatedLocationDataSource` to get location updates.
+4. On location updates, store that location, display the location as a point on the map, and recreate the route polyline.:q
+
+## Relevant API
+
+* Location.LocationDisplayAutoPanMode
+* Location.SimulatedLocationDataSource
+* Location.SimulatedLocationDataSource.LocationChanged
+* MapView.LocationDisplay
```

Comment:
```suggestion
* LocationDisplayAutoPanMode
* SimulatedLocationDataSource
* LocationDataSource
* LocationDisplay
* PolylineBuilder
* SimpleLineSymbol
* SimpleMarkerSymbol
* SimpleRenderer
```

Add these to the relevant API, and the metadata.json

Reply:
any reason LocationDataSource should be in relevant API? Since we are using `SimulatedLocationDataSource` here

Reply:
`SimulatedLocationDataSource` is a subclass of `LocationDataSource`, and it's also specified in the common design for the sample.